spacy/compat.py

```diff
@@ -23,6 +23,11 @@
 except ImportError:
     cupy = None
 
+if sys.version_info[:2] >= (3, 8):  # Python 3.8+
+    from typing import Literal, Protocol, runtime_checkable
+else:
+    from typing_extensions import Literal, Protocol, runtime_checkable  # noqa: F401
+
```

Comment:
This shouldn't be necessary since we've dropped python 3.7 already.

Reply:
Fixed.